ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-internet-exit.j2

```diff
@@ -0,0 +1,49 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - router internet exit #}
+{% if router_internet_exit is arista.avd.defined %}
+
+### Router Internet Exit
+
+#### Exit Group Summary
+
+| Exit Group | Local Connection | Fib Default | Policy | Policy Exit Group |
+| ---------- | ---------------- | ----------- | ------ | ----------------- |
+{%     if router_internet_exit.exit_groups is arista.avd.defined %}
```

Comment:
Make sure to check this if statement BEFORE you render the title above - no exit_group -> no title

Reply:
Done!